app/app/settings.py

```diff
@@ -408,6 +408,15 @@
 COLO_ACCOUNT_ADDRESS = env('COLO_ACCOUNT_ADDRESS', default='')  # TODO
 COLO_ACCOUNT_PRIVATE_KEY = env('COLO_ACCOUNT_PRIVATE_KEY', default='')  # TODO
 
+IPFS_HOST = env('IPFS_HOST', default='ipfs')
```

Comment:
shouldnt this default to infura?

Reply:
We can definitely default to infura, if you want.  But I figured we could default to the local docker container of IPFS (all ipfs data will persist just as long as the DB data, so this should suffice and reduce latency) and use our hosted IPFS nodes as the primary on staging/live.